selenium/chrome/local/Dockerfile

```diff
@@ -45,11 +45,14 @@ RUN \
          lsb-base \
```

Comment:
Could you create a new docker file for beta? Similarly we will have another one for dev when we build images for dev. Might need to change the scripts to pass browser channel (beta or dev) as a parameter and according use the corresponding docker file.

Reply:
I have these things in mind. I'll go back to them after APT is done. This Draft PR is just a reminder of the research done so far.